README.md

```diff
@@ -8,34 +10,39 @@
     ```
     docker build -t run-lab -f run-lab/Dockerfile .
     ```
-    run-lab image is a dockerized version of `run-lab/coursera-lab.py` script. `run-lab/coursera-lab.py` script simulates Coursera Labs docker build and run procedures.
-    
+    The `run-lab` image is a Dockerized version of `run-lab/coursera-lab.py` script. `run-lab/coursera-lab.py` script simulates Coursera Labs Docker build and run procedures.
+
 3. **Create a folder with your custom image files and `manifest.json`**
 
-    We strongly recommend to use our example images `jupyter/datascience-notebook`, `jupyter/scipy-notebook` or `jupyter/tensorflow-notebook` as a base of your image. 
-    Coursera labs script uses `manifest.json` to read configurations specific to your image. Details about `manifest.json` can be found [here](#manifestjson)
-    
-4. **Build and Test your custom image locally**
+    The folders `jupyter/datascience-notebook`, `jupyter/scipy-notebook` and `jupyter/tensorflow-notebook` include the files that build our base images. Please feel free to use these folders to test out this build flow.
+
+    We also *strongly* recommend using the provided images as the basis for your custom images.
+
+    Additionally, Coursera Labs script uses `manifest.json` to read configurations specific to your image. Details about `manifest.json` can be found [here](#manifestjson).
```

Comment:
I see that you have made update to the link `#manifestjson` to `#image-configurations-using-manifestjson`

Reply:
Good catch! I'll update now.